docker/src/main/docker/mysql/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2'
+services:
+  avatica:
+    ports:
+      - "8765:8765"
+    links:
+      - mysql
+    image: avatica-mysql-server
+    command: "-u 'jdbc:mysql://mysql:3306/avatica' -p 8765"
+    depends_on:
+      - mysql
+  mysql:
+    image: mysql:5.7
+    ports:
+      - "3306:3306"
```

Comment:
Hmm, that's a good reason to not expose the ports. I was thinking that I was being "helpful" by exposing the backend db, but, like you said, we don't really expect users to actually do this.